R/constants.R

```diff
@@ -152,6 +152,8 @@ ELECTRICITY_INPUT_FUELS<- c( "biomass", "coal", "gas", "refined liquids" )
 # and should contain all historical years used by other modules
 socioeconomics.MADDISON_HISTORICAL_YEARS <- seq(1700, 1900, 50) # Years for which to use Maddison data
 socioeconomics.UN_HISTORICAL_YEARS <- c(1950, 1971:2010) # Years for which to use UN data
+socioeconomics.AGG_BASE_YEAR <- 1950 # Year used to generate ratios for region -> country downscaling
+socioeconomics.FINAL_HIST_YEAR <- 2010 # Final historical year
 socioeconomics.BASE_POP_SCEN <- "SSP2"
```

Comment:
Yes, looks like the socioeconomics constants are in two spots. We should put them in the same place and decide on a format. I've been using the "socioeconomics.XXX" format. This is something we might want to get consistency on across all the constants (the AGLU constants sometimes have the "aglu." prefix and sometimes don't, same for socioeconomics, I guess.)